doc/source/develop-arguments.rst

```diff
@@ -0,0 +1,101 @@
+Command-line arguments
+================================================================================
+
+To take full advantage of Pypiper (make your pipeline recoverable, etc.), you may choose to add command-line options to your pipeline that pypiper understands. Pypiper uses the typical Python `argparse module <https://docs.python.org/2/library/argparse.html>`_ to define command-line arguments to your pipeline.
+
+You can use an ArgumentParser as usual, adding whatever arguments you like. Then, you add Pypiper args to your parser with the function ``add_pypiper_args()``, and pass the parser to your ``PipelineManager``, like this:
+
+.. code-block:: python
+
+    import pypiper, os, argparse
+    parser = ArgumentParser(description='Write a short description here')
+
+    # add any custom args here
+    # e.g. parser.add_argument('--foo', help='foo help')
+
+    # once you've established all your custom arguments, we can add the default
+    # pypiper arguments to your parser like this:
+
+    parser = pypiper.add_pypiper_args(parser)
+    
+    # Then, pass this parser along to the PipelineManger
+
+    pipeline = pypiper.PipelineManager(name="my_pipeline", outfolder="out", \
```

Comment:
It looks like the argument to `args` should be a `Namespace`, but I think thathere a `ArgumentParser` is pased.